### PR TITLE
Add more tests and use the OpenStack testing framework

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs please review

File: .kitchen.cloud.yml [hunk 1]
```diff
@@ -74,6 +77,8 @@ suites:
     run_list:
       - role[openstack_ops_database]
       - role[openstack_controller]
+      - recipe[openstack-integration-test::setup]
+      - recipe[scl]
```
Comment (jordane) [inline .kitchen.cloud.yml]: This cookbook should be in the Berksfile, since it isn't in metadata.

Comment (jordane) [inline .kitchen.cloud.yml]: Oh, it is in the metadata. If you're only calling it from .kitchen.cloud.yml, and not anywhere in `recipes/`, it shouldn't be in metadata.

Comment (ramereth) [inline .kitchen.cloud.yml]: So should I just add it to Berksfile and remove it from metadata?

Comment (jordane) [inline .kitchen.cloud.yml]: Yes please. It only belongs in the metadata if the cookbook uses it directly. If TK uses it, just the berksfile.

**Comment by jordane** [comment]
+1 other than my small nit

**Pull request merged**